examples/geoObject/geoObject.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+import { Entity } from "../../src/og/entity/Entity.js";
+import { EntityCollection } from "../../src/og/entity/EntityCollection.js";
+import { Globe } from "../../src/og/Globe.js";
+import { XYZ } from "../../src/og/layer/XYZ.js";
+import { GlobusTerrain } from "../../src/og/terrain/GlobusTerrain.js";
+import { LonLat } from '../../src/og/LonLat.js';
+import * as utils from "../../src/og/utils/shared.js";
+
+let COUNT = 10,
+    ENTITY = {},
+    ENTITY_DEFAULT_OPTIONS = {
+        farmplane: (options) => {
+            options.geoObject.scale = 100;
+            options.geoObject.yaw = -50;
+            return {
+                ...options,
+                lonlat: [rnd(-180, 180), rnd(-180, 180), 20000]
+            };
+        },
+        satellite: (options) => {
+            return {
+                ...options,
+                lonlat: [rnd(-180, 180), rnd(-180, 180), 2000000]
+            };
+        },
+        airplane: (options) => {
+            options.geoObject.yaw = 75;
+            return {
+                ...options,
+                lonlat: [rnd(-180, 180), rnd(-180, 180), 20000]
+            };
+        }
+    };
+
+const div = document.createElement('div');
+div.style.setProperty('display', 'flex');
+div.style.setProperty('flex-direction', 'column');
+div.style.setProperty('position', 'absolute');
+div.style.setProperty('top', '10px');
+div.style.setProperty('left', '10px');
+div.style.setProperty('color', 'white');
+div.style.setProperty('background', `rgba(0, 0, 0, .7)`);
+document.body.appendChild(div);
+
+const span = document.createElement('span');
+span.setAttribute('id', 'instance-count');
+span.innerText = `Instance count: ${COUNT}`;
+div.appendChild(span);
+
+const range = document.createElement('input');
+range.setAttribute('type', 'range');
+range.setAttribute('min', '1');
+range.setAttribute('value', '1');
+range.setAttribute('max', '5000');
+range.addEventListener('input', () => {
+    COUNT = parseInt(range.value);
+});
+div.appendChild(range);
+
+function rnd(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+let globus = new Globe({
+    target: "globus",
+    name: "Earth",
+    layers: [
+        new XYZ("OpenStreetMap", {
+            isBaseLayer: true,
+            url: "//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+            visibility: true,
+            attribution: "Data @ OpenStreetMap contributors, ODbL"
+        })
+    ],
+    terrain: new GlobusTerrain()
+});
+let colors = ["red", "orange", "yellow", "green", "lightblue", "darkblue", "purple"];
+
+let geoObjects = new EntityCollection({
+    entities: [],
+    scaleByDistance: [600000, 24000000, 10000000000]
+});
+
+const instancesTypes = [...new Array(80).fill('airplane'), ...new Array(20).fill('farmplane'), 'satellite'];
```

Comment:
I do not like this iteration logic. I suggest replace it with something more convenient:
```
[
  {
      instanceName: "airplane",
      quantity: 80 
  }, {
      instanceName: "farmplane",
      quantity: 20 
  }.
  ...
]
```